docs/config/index.md

```diff
@@ -121,3 +121,19 @@ export default defineConfig(({ mode }) => {
   }
 })
 ```
+
+## VScode Config
+
+Vite 6 writes the generated temporary configuration file to the `node_modules/.vite-temp` folder, a file not found error will occur when setting breakpoint debugging in the vite config file. Adding the following configuration to the vscode configuration file can solve the problem.
```

Comment:
```suggestion
With a default `--configLoader bundle`, Vite 6 writes the generated temporary configuration file to the `node_modules/.vite-temp` folder, a file not found error will occur when setting breakpoint debugging in the vite config file. Adding the following configuration to the vscode configuration file can solve the problem.
```

Reply:
Thanks for the suggestion, it looks great. 👍